src/opera/commands/deploy.py

```diff
@@ -19,29 +19,30 @@ def add_parser(subparsers):
     parser.set_defaults(func=deploy)
 
 
+# pylint: disable=unused-argument
 def deploy(args):
-    csar.save(args.name, args.csar.name)
+    # csar.save(args.name, args.csar.name)
     status = 0
 
-    print("Loading service template ...")
+    logger.info("Loading service template ...")
     try:
-        ast = tosca.load(Path.cwd(), PurePath(args.csar.name))
-        print("=======================")
-        tmpl = template.build(ast)
-        print(tmpl)
+        # ast = tosca.load(Path.cwd(), PurePath(args.csar.name))
```

Comment:
Is the deploy now fully disabled? Or have I missed something and it's started from somewhere else?

Reply:
I haven't touched the deploy and undeploy code paths, apart from commenting out code so the parser and linter don't complain. The refactor before mine was only partial, and left the two in a non-working state; see 5cad99bbcea939df40d0b0cbedef043fac5e179c. Perhaps @tadeboro has some plans for this.